src/server/scripts/Northrend/Gundrak/boss_moorabi.cpp

```diff
@@ -74,24 +82,32 @@ class boss_moorabi : public CreatureScript
             void Initialize()
             {
                 _transformed = false;
+                events.ScheduleEvent(EVENT_PHANTOM, Seconds(21), 0, PHASE_INTRO);
```

Comment:
Initialize() we use to initialize local variables.
Change this ScheduleEvent call for Reset()